### PR TITLE
[DOCS] add vetiver to spacy universe

### PR DESCRIPTION
**Pull request by isabelizimm**
Hey all--

After [soliciting feedback](https://github.com/explosion/spaCy/discussions/12475) and incorporating spaCy support in a package I maintain, [vetiver](https://vetiver.rstudio.com/), I would love to have this project advertised with other spaCy tools!

## Description
Add new project (vetiver) to list of projects in spacy universe

### Types of change
Addition to website

## Checklist

- [X] I confirm that I have the right to submit this contribution under the project's MIT license.
- [X] I ran the tests, and all new and existing tests passed.
- [X] My changes don't require a change to the documentation, or if they do, I've added all required information.

### CODE REVIEW

**Comment by bdura** [comment]
Hello @isabelizimm, thanks again for the submission!

I took the example for a quick spin, it was extremely easy to get a pipeline running. However the auto-generated API documentation does not explicit the input schema for the `predict` endpoint. I tried a few guesses but none of them worked. Do you think you could add more documentation on this?

**Comment by isabelizimm** [comment]
You're welcome! Yep, I can add those updates to make it more clear how to use the `predict` function in a spaCy workflow. I'll plan to update both the API docs, and possibly the example for the universe page as well. 👍

File: website/meta/universe.json [hunk 1]
```diff
@@ -4119,6 +4119,33 @@
...
+        {
+            "id": "vetiver",
+            "title": "Vetiver",
+            "slogan": "Version, share, deploy, and monitor models.",
+            "description": "The goal of vetiver is to provide fluent tooling to version, deploy, and monitor a trained model. Functions handle creating model objects, versioning models, predicting from a remote API endpoint, deploying Dockerfiles, and more.",
+            "github": "rstudio/vetiver-python",
+            "pip": "vetiver",
+            "code_example": [
+                "from vetiver import VetiverModel, VetiverAPI",
+                "",
```
Comment (bdura) [inline website/meta/universe.json:4132]:
```suggestion
                "import spacy",
                "from vetiver import VetiverModel, VetiverAPI",
                "",
                "# If you use this model, you'll need to download it first:",
                "# python -m spacy download en_core_web_sm",
                "nlp = spacy.load('en_core_web_md')",
```

I suggest adding a pipeline definition to make the example work out-of-the-box.

**Review by bdura — COMMENTED** [review]
Hello @isabelizimm,

I made a suggestion to make the example work out-of-the-box by adding a pipeline definition. And if you have some more documentation on how to query the API, it might help users if you added a link within the description.

Otherwise everything looks good to me!

File: website/meta/universe.json [hunk 2]
```diff
@@ -4119,6 +4119,37 @@
...
+            "slogan": "Version, share, deploy, and monitor models.",
+            "description": "The goal of vetiver is to provide fluent tooling to version, deploy, and monitor a trained model. Functions handle creating model objects, versioning models, predicting from a remote API endpoint, deploying Dockerfiles, and more.",
+            "github": "rstudio/vetiver-python",
+            "pip": "vetiver",
+            "code_example": [
+                "import spacy",
+                "from vetiver import VetiverModel, VetiverAPI",
+                "",
+                "# If you use this model, you'll need to download it first:",
+                "# python -m spacy download en_core_web_sm",
```
Comment (svlandeg) [inline website/meta/universe.json:4135]:
```suggestion
                "# python -m spacy download en_core_web_md",
```

**Review by svlandeg — APPROVED** [review]
I think we can merge this once the CI goes green. Thanks again for the contribution!

**Comment by isabelizimm** [comment]
Thanks for merging! I have a PR to refresh vetiver's docs that will add more info on how to query the `predict` endpoint 🙌

**Pull request merged**